lib/sna/auth/email.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Sna.Auth.Mail do
```

Comment:
Shouldn't the file be named `lib/sna/auth/mail.ex` instead ?

Reply:
yes, mistake, I will correct that.